python/doc/examples/numerical_methods/estimate_integral_Clenshaw_Curtis.ipynb

```diff
@@ -0,0 +1,131 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Estimate an integral"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this basic example we are going to evaluate an integral of the form.\n",
```

Comment:
@mbaudin47 has made the case that the basic/advanced examples distinction is unnecessary. See PR #1306 which, among other things, removes this distinction in all existing notebooks.
Moreover, the point should be made to a colon:
```suggestion
    "In this example we are going to evaluate an integral of the form:\n",
```